src/utils/date.spec.ts

```diff
@@ -11,6 +11,7 @@ describe("inRange", () => {
   it("returns true if no min/max", () => {
     expect(inRange(new Date())).toEqual(true);
   });
+
```

Comment:
Since there were no relevant code changes here, can you submit this separately as a `style` PR?